README.md

```diff
@@ -37,6 +37,7 @@ Born in 2016 to power the Django Channels project, ASGI and its ecosystem have b
 _Frameworks for building ASGI web applications._
 
 - [Channels](https://channels.readthedocs.io/en/latest/) - Asynchronous support for Django, and the original driving force behind the ASGI project. Supports HTTP and WebSockets with Django integration, and any protocol with ASGI-native code.
+- [Django](https://docs.djangoproject.com/en/3.0/topics/async/) - From version 3.0, Django has included native ASGI support.
```

Comment:
```suggestion
- [Django](https://docs.djangoproject.com/en/3.0/topics/async/) - The web framework for perfectionists with deadlines. Has native ASGI support since version 3.0.
```